mslice/plotting/plot_window/plot_options.py

```diff
@@ -249,6 +249,8 @@ class LegendAndLineOptionsSetter(QtWidgets.QWidget):
 
     def __init__(self, line_options, color_validator):
         super(LegendAndLineOptionsSetter, self).__init__()
+        self.line_options = line_options
```

Comment:
I don't think we need this?

Reply:
Yes, we don't. I was testing something there